src/lib/components/injectors.ts

```diff
@@ -43,6 +43,12 @@ export function CustomElement(): any {
         if (!inPageContext()) {
             return;
         }
+
+        if (customElements.get(target.tag())) {
```

Comment:
Note: This is put here since FF emits warnings on re-defining a tag while Chrome ignores